### PR TITLE
Setup continuous testing using GitHub Actions

### PR DESCRIPTION
**Pull request by juarezr**
### Features

* [x] Setup
  * [x] Install Rust stable toolchain
  * [x] Install Firebird cli and development library
  * [x] Install Firebird in a container for running tests
* [x] Testing
  * [x] Run cargo check
  * [x] Run cargo test
  * [x] Run cargo clippy
  * [x] Run cargo fmt --check
  * [x] Create source coverage
  * [x] Upload coverage to Coveralls
* [x] Next steps
  * [ ] Run test with Firebird 2.5-ss, 3.0 and 4.0
  * [ ] Run on Windows
  * [ ] Run on Linux/ARM
  * [ ] Run on Linux/ARM/LittleEndian
  * [ ] Release workflow to crates.io and rust docs.
* [x] Status
  * [x] Installing, building, testing and linting working.
  * [ ] Coverage failing on cc linking (needs help...)
  * [x] Ready for review
  * [x] Ready for merge
  * [ ] Next steps on further PRs

### CODE REVIEW

**Comment by fernandobatels** [comment]
I think that we can convert this PR to a draft until the completion of your description points

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Rust Firebird Client 
 
+![Build](https://github.com/juarezr/rsfbclient/workflows/testing_changes/badge.svg)
```
[the remark this answers is not in the document]
Comment (juarezr) [reply README.md:3]: Good catch!

**Comment by juarezr** [comment]
> I think that we can convert this PR to a draft until the completion of your description points

I'm mostly done with this PR except if someone request changes in review. 

Also I was planning to develop further features in new PRs.

So maybe it's not worth changing this PR to draft.

But I'm open to hear what's better fits here.

**Comment by fernandobatels** [comment]
> > I think that we can convert this PR to a draft until the completion of your description points
> 
> I'm mostly done with this PR except if someone request changes in review.
> 
> Also I was planning to develop further features in new PRs.
> 
> So maybe it's not worth changing this PR to draft.
> 
> But I'm open to hear what's better fits here.

Ok. I will merge this PR and open a new issue with the remaining points.

**Pull request merged**